RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Row/NonFungbileAssetRow+Reducer.swift

```diff
@@ -88,7 +88,7 @@ extension NonFungibleAssetList {
 						state.tokens = .init(repeating: .init(repeating: .loading, count: State.pageSize), count: fullPagesCount)
 						/// The number of items to add to the last page
 						let remainder = state.resource.nonFungibleIdsCount % State.pageSize
-						if fullPagesCount > 1, remainder > 0 {
```

Comment:
Bad implementation by me... We do need to add the remainder items if there is at least 1 page, not at least 2...